web.sh

```diff
@@ -1,28 +1,28 @@
 #!/usr/bin/env bash
 
-# Install command-line tools using Homebrew.
+# Install tooling needed for Front End developers.
 
 # Ask for the administrator password upfront.
 sudo -v
 
 # Keep-alive: update existing `sudo` time stamp until the script has finished.
 while true; do sudo -n true; sleep 60; kill -0 "$$" || exit; done 2>/dev/null &
 
-# Check for Homebrew,
-# Install if we don't have it
-if test ! $(which brew); then
-  echo "Installing homebrew..."
-  ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-fi
+# Download and install nvm - Node version manager
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash
 
-# Make sure we’re using the latest Homebrew.
-brew update
```

Comment:
This is needed in all scripts. Why did we remove it?

Reply:
Should we install `nvm` from brew?

Reply:
I didn't check for `nvm` formulae for `homebrew` in the first place. There is one, so I'll use brew and revert `homebrew` check.